src/handlers.jl

```diff
@@ -230,7 +230,7 @@ end
 """
     Escalator(fmt=DefaultFormatter(); level="warn", levels=nothing)
 
-Escalates any logs it sees above a certain `level` to [`EscalationError`s](@ref).
+Escalates any logs it sees above a certain `level` and throws an `EscalationError`.
```

Comment:
Should the ref be removed?

Reply:
Yes, because documenter is a pain and keeps complaining that it can't find the reference...